environments/nodejs/package.json

```diff
@@ -14,13 +18,10 @@
   },
   "dependencies": {
```

Comment:
I didn't get any issue so far.
I think it will causes confusion for new users if we didn't remove the unused modules and the number may grow up as nobody why those modules existing for.

Reply:
You wouldn't find an issue unless you were using those dependencies; if you were using them your functions would break.  So we can't remove them without deprecating them first.  Let's get there slowly; keep the old dependencies, then we'll deprecate at some point, and remove them later.

I understand your concern about the module list growing, I don't think it's too big yet but we should watch out for that; and your change certainly helps with that problem.